assets/web/header.html

```diff
@@ -5,6 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Sunshine</title>
+    <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
+    <link href="/third_party/font-awesome.5.15.4.all.min.css" rel="stylesheet">
```

Comment:
This file does now work when saved locally. We either need to refer to "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" OR use the entire fond structure locally.

Ideally would serve the fonts folder (doesn't require internet access).
![image](https://user-images.githubusercontent.com/42013603/152663943-3b41dde0-3f06-4e04-8c33-27acfd329fe2.png)


Do you know how to serve an entire directory (recursively) with this web server?

Reply:
I totaly agree with you if the goal is also to have a totaly offline solution, but we have to go step by step. As far as I know Simple-Web-Server doesn't support file serving and this will be an issue.

I can suggest you, for the moment, using an external cdn and forget about file serving.

Concerning webui and httpd from my point of view we have to: 
1. Take rough decissions and clear requirements about the backend supporting this configuration webui.
2. Implement the new backend and make current webui working with it.
3. Then refactor the configuration webui to be totaly available offline.

Reply:
Perhaps I used the incorrect terminology. I think it must support file "serving" or the html pages wouldn't load. Aren't the `server.resource` lines in `confighttp` serving the resources?

I'll see if Elix has any suggestions as well. Certainly using the CDN and web resources would be the easiest method.

Also, thank you for reviewing!

Reply:
Ah yes exactly. May be you could try something like this (don't know the exact syntaxe):

´´´cpp
server.resource["^/static/images/(.+)$"]["GET"] = getImages;
server.resource["^/static/css/(.+)$"]["GET"] = getCss;
server.resource["^/static/fonts/(.+)$"]["GET"] = getFonts;
´´´

or serve all static files:

´´´cpp
server.resource["^/static/(.+)$"]["GET"] = getStaticRessources;
´´´

Then into `getStaticRessources` return `WEB "staticresourcepath"` I think return everything as binary should work, but you'll have to detect `Content-Type` accordingly.

Sorry it's been a long time that I haven't code 😄. Some web frameworks provide a file serve functionality by default. With something like `server.serve_static("staticefilesdirectory");`